app/src/main/java/org/stepic/droid/util/ProfileSettingsHelper.kt

```diff
@@ -63,11 +59,6 @@ fun ProfileSettingsViewModel?.clickProfileSettings(activity: Activity,
             screenManager.showDownloads(activity)
         }
 
-        notificationsTitleRes -> {
-            analytic.reportEvent(Analytic.Screens.USER_OPEN_NOTIFICATIONS)
```

Comment:
it should be added [here](https://github.com/StepicOrg/stepik-android/blob/3d98a1752ae91674de263fd98a62a8f702a087f5/app/src/main/java/org/stepic/droid/ui/activities/MainFeedActivity.kt#L234)